bot/test-i9n/StateRelayerBot.unit.ts

```diff
@@ -72,59 +71,61 @@ describe('State Relayer Bot Tests', () => {
       testGasCost: false,
       envNetwork: EnvironmentNetwork.LocalPlayground,
       urlNetwork: '',
-      contractAddress: proxy.address,
+      contractAddress: await proxy.getAddress(),
       signer: bot,
     });
 
+    const client = getWhaleClient('', EnvironmentNetwork.LocalPlayground);
+    const testPoolPairData = await client.poolpairs.list(200);
     if (output !== undefined) {
-      const { dexInfoTxReceipt, masterDataTxReceipt, vaultTxReceipt, burnTxReceipt } = output;
+      const { dexInfoTxReceipt, masterDataTxReceipt, vaultTxReceipt } = output;
       expect(dexInfoTxReceipt).toBeUndefined();
       expect(masterDataTxReceipt).toBeUndefined();
       expect(vaultTxReceipt).toBeUndefined();
-      expect(burnTxReceipt).toBeUndefined();
     }
 
-    const receivedBurnedInfo = await proxy.getBurnedInfo();
-    expect(receivedBurnedInfo[1].fee.toString()).toEqual(expectedBurnedInfo.fee);
-    expect(receivedBurnedInfo[1].auction.toString()).toEqual(expectedBurnedInfo.auction);
-    expect(receivedBurnedInfo[1].payback.toString()).toEqual(expectedBurnedInfo.payback);
-    expect(receivedBurnedInfo[1].emission.toString()).toEqual(expectedBurnedInfo.emission);
-    expect(receivedBurnedInfo[1].total.toString()).toEqual(expectedBurnedInfo.total);
-
     // Checking the /dex/dex-pair info
     const dETH = await proxy.getDexPairInfo('dETH-DFI');
-    expect(dETH[1].primaryTokenPrice.toString()).toEqual(Object.values(expectedPairData['dETH-DFI'])[0]);
-    expect(dETH[1].volume24H.toString()).toEqual(Object.values(expectedPairData['dETH-DFI'])[1]);
-    expect(dETH[1].totalLiquidity.toString()).toEqual(Object.values(expectedPairData['dETH-DFI'])[2]);
-    expect(dETH[1].APR.toString()).toEqual(Object.values(expectedPairData['dETH-DFI'])[3]);
-    expect(dETH[1].firstTokenBalance.toString()).toEqual(Object.values(expectedPairData['dETH-DFI'])[4]);
-    expect(dETH[1].secondTokenBalance.toString()).toEqual(Object.values(expectedPairData['dETH-DFI'])[5]);
-    expect(dETH[1].rewards.toString()).toEqual(Object.values(expectedPairData['dETH-DFI'])[6]);
-    expect(dETH[1].commissions.toString()).toEqual(Object.values(expectedPairData['dETH-DFI'])[7]);
-    expect(dETH[1].decimals.toString()).toEqual(Object.values(expectedPairData['dETH-DFI'])[8]);
+    const expectedDexInfo = tranformPairData(testPoolPairData, mockedStatsData, mockedDexPricesData);
+    const lastETHDFIInfo = expectedDexInfo.dexInfo[expectedDexInfo.dex.indexOf('dETH-DFI')];
+    // for sure both two sides have the same type as bigint
+    expect(dETH[1].primaryTokenPrice).toStrictEqual(Object.values(lastETHDFIInfo)[0]);
```

Comment:
`Object.values(lastETHDFIInfo)[0]` - why not `lastETHDFIInfo.primaryTokenPrice`?